grouper/fe/handlers.py

```diff
@@ -732,6 +743,16 @@ def get(self):
             .filter(Group.enabled == True)
             .order_by(Group.groupname)
         )
+        if audited_only:
+            audited_permissions = (
+                self.session.query(Permission)
+                .filter(Permission._audited == True).all()
+            )
+            groups = groups.filter(
```

Comment:
This won't catch inheritance, will it? I.e. permission on group `x` makes it audited, but `y` is a member of `x`, so `y` should also be shown as audited.